angular/src/environments/environment.prod.ts

```diff
@@ -20,5 +20,5 @@ export const environment: IEnvironment = {
   instanceUrl: 'https://www.blockcore.net/',
   instanceExplorerUrl: 'https://explorer.blockcore.net',
   networks: [] as string[] // Returns all
-  // networks: ['STRAX', 'CRS', 'TSTRAX', 'TCRS', 'IMPLX', 'MOL', 'CITY', 'CY', 'RSC', 'SBC', 'X42']
+   //networks: ['STRAX', 'CRS', 'TSTRAX', 'TCRS', 'IMPLX', 'MOL', 'CITY', 'CY', 'RSC', 'SBC', 'X42','XRC']
```

Comment:
Please remove this, no networks should be defined here, that will ensure all networks are always loaded.

Reply:
Ok i removed whole line. :) Thank you :)